crates/yakui-winit/src/keys.rs

```diff
@@ -1,100 +1,101 @@
-use winit::event::VirtualKeyCode;
 use yakui_core::input::{KeyCode, Modifiers};
 
-pub fn from_winit_key(key: VirtualKeyCode) -> Option<KeyCode> {
+pub fn from_winit_key(key: winit::keyboard::KeyCode) -> Option<KeyCode> {
+    use winit::keyboard::KeyCode as WinitKey;
```

Comment:
I believe this is a change in behavior.

- winit's `VirtualKeyCode` became `Key`, referring to a key that should have the given text on its keycap.
- winit's `ScanCode` became `KeyCode`, used here, which refers to a physical location of a key based on its label on a US keyboard.

For what yakui does, I think that `Key` is the more appropriate default. Using an example of a key combo we don't actually currently have, pressing `ctrl-a`/`cmd-a` should select all text, regardless of where `a` is on the user's keyboard. However, using `KeyCode` and physical keys, that would mean that users with the AZERTY layout would instead press `ctrl-q`/`cmd-q` to select all text.

I think that would be odd, but I'm interested to hear your opinion on this.